docs/quickstart.rst

```diff
@@ -142,6 +161,20 @@ Open Telemetry
 
 See :ref:`otel` to configure metrics, traces and logging.
 
+Views
+-----
+
+CSRF settings
+"""""""""""""
+
+In order to properly handle the CSRF protection, add the following settings to your Django configuration:
+
+- ``LOGIN_URLS`` specifies the URL(s) where users will be redirected if
+  authentication is required.
+- ``CSRF_FAILURE_VIEW`` defines the view that will be called when CSRF validation fails.
+
+The provided ``maykin_common.views.csrf_failure`` offers a standardized failure page. See :mod:`maykin_common.views`
```

Comment:
```suggestion
```
it doesn't really do anyting in "providing a standardized failure page", all it does is ignore the failure if the user double clicked on the login button causing a race condition to be triggered because on login the CSRF token is cycled.